docs/metrics/lead-cycle-time.md

```diff
@@ -0,0 +1,33 @@
+# Lead and cycle time metrics
+
+Monitoring you team's lead time and cycle time allows you to understand if you're improving the ability to deliver value to customers.
+
+![Lead time versus cycle time](images/lead-cycle-time.png)
+
+-   [Lead time](#lead-time)
+-   [Cycle time](#cycle-time)
+
+!!! note
+    Pulse calculates lead time and cycle time based on the state changes of issues **that are already closed** in Jira. This means that:
```

Comment:
Not sure if we should mention that we will consider all closed issues, independently of resolution.

Reply:
This means that we also consider issues with [resolutions such as **Won't do** and **Duplicate**](https://support.atlassian.com/jira-cloud-administration/docs/what-are-issue-statuses-priorities-and-resolutions/#Issue-resolutions) in the calculation, correct?

Reply:
Yes, that's correct. For now we don't distinguish between resolutions.